sdl2-sonic-drivers/src/files/dmx/MUSFile.cpp

```diff
@@ -59,14 +61,9 @@ namespace files
             readTrack();
         }
 
-        std::shared_ptr<audio::MIDI> MUSFile::getMIDI(std::shared_ptr<files::dmx::OP2File> op2file) noexcept
-        {
-            return convertToMidi(op2file);
-        }
-
         std::shared_ptr<audio::MIDI> MUSFile::getMIDI() noexcept
         {
-            return convertToMidi(nullptr);
+            return convertToMidi();
```

Comment:
this could be cached here in a shared ptr so the coversion will happen only once.